tools/mpy-tool.py

```diff
@@ -428,6 +428,9 @@ def read_bytecode_qstrs(file, bytecode, ip):
         ip += sz
 
 def read_raw_code(f):
+    kind = read_uint(f)
```

Comment:
Well, this doesn't correspond to C code, which reads a byte.

Reply:
Now fixed.